src/serialized-analysis.ts

```diff
@@ -44,7 +44,10 @@ export interface SourceLocation {
 }
 
 export interface AnalyzedPackage {
```

Comment:
Renamed `serialized-analysis.ts` to `elements-metadata.ts` (and likewise its test).

What should we rename `AnalyzedPackage` to?